skcosmo/pcovr/pcovr.py

```diff
@@ -144,8 +144,7 @@ def _compute_Yhat(self, X, Y, Yhat=None, W=None):
                 Yhat = X @ W
 
         elif W is None:
-            W = np.linalg.pinv(np.dot(X.T, X), rcond=self.regularization)
-            W = np.linalg.multi_dot([W, X.T, Y])
+            W = np.linalg.lstsq(X, Y, rcond=self.regularization)
```

Comment:
```suggestion
            W = np.linalg.lstsq(X, Y, rcond=self.regularization)[0]
```

`np.linalg.lstsq` returns a tuple (weights, residuals, rank, singular values); we just want the weights.